java/ql/test/TestUtilities/InlineMadTest.qll

```diff
@@ -0,0 +1,65 @@
+import java as J
+
+private signature module InlineMadTestLangSig {
+  /**
+   * A base class of callables for modeling.
+   */
+  class Callable;
+
+  /**
+   * Gets a relevant code comment for `c`, if any.
+   */
+  string getComment(Callable c);
+}
+
+private module InlineMadTestImpl<InlineMadTestLangSig Lang> {
+  private class Callable = Lang::Callable;
+
+  signature module InlineMadTestConfigSig {
+    /**
+     * Gets the kind of a captured model.
+     */
+    string getKind();
+
+    /**
+     * Gets a captured model for `c`, if any.
+     */
+    string getCapturedModel(Callable c);
```

Comment:
(trivial nitpick) We may as well put them in alphabetical order.
```suggestion
    /**
     * Gets a captured model for `c`, if any.
     */
    string getCapturedModel(Callable c);

    /**
     * Gets the kind of a captured model.
     */
    string getKind();
```

Reply:
Thank you! I will do that in the follow up when moving this into the shared library!